contracts/contracts/AddressRegistry.sol

```diff
@@ -1,4 +1,4 @@
-//SPDX-License-Identifier: MIT
```

Comment:
Why were these changed? Our repo and the code within is on the [MIT License](https://github.com/web3well/bls-wallet/blob/main/LICENSE) (not including our deps)

Reply:
Core contracts are Unlicense, repo is MIT. Will discuss with James. Fine as is for now.